mod/aalborg_theme/views/default/css/elements/typography.php

```diff
@@ -115,6 +115,10 @@
 	font-style: italic;
 }
 
+.elgg-subtext-link {
```

Comment:
I vaguely remember that effected something else, but I guess 2.0 would be the time to do it.